components/feature/addons/src/main/java/mozilla/components/feature/addons/AddonManager.kt

```diff
@@ -70,7 +70,8 @@ class AddonManager(
                 .map { addon -> addon.filterTranslations(locales) }
                 .map { addon ->
                     installedExtensions[addon.id]?.let {
-                        addon.copy(installedState = it.toInstalledState())
+                        addon.copy(installedState = it.toInstalledState(),
+                                version = it.toInstalledState().version.ifEmpty { addon.version })
```

Comment:
So is this not where we update version after updating?

Reply:
As we are not re-querying the AMO endpoints after update, we are using the info that Gecko has about the installed add-on, this way we are always going to have the latest version without without hitting the network.

Reply:
Okay cool, makes sense to me 👍